src/env.ts

```diff
@@ -99,12 +98,10 @@ if (ENV.DEBUG) {
   
   // Only show parts of sensitive values
   const sanitizedEnv = sanitizeSecrets({
-    API_KEY: ENV.API_KEY,
-    API_SECRET: ENV.API_SECRET
+    API_KEY: ENV.API_KEY
   });
   
   console.error(`API Key: ${sanitizedEnv.API_KEY}`);
```

Comment:
If we just have to sanitize the secrets anyway... why bother logging this? Maybe we avoid all the sanitization stuff, and just print status and updates, rather than the actual sanitized key?

Reply:
makes sense: [ecb5d45](https://github.com/recallnet/trading-simulator-mcp/pull/2/commits/ecb5d451b20e503fa2ad54c679c7f1974cced950)